wgsl/index.bs

```diff
@@ -3142,36 +3109,451 @@ TODO: deduplicate these tables
   <tr><td>*e* : *T*, *T* is *FloatVec*<td>`isNormal(e)` : bool<*N*>, where *N = Arity(T)*<td>OpIsNormal, or emulate
 </table>
 
+## Float built-in functions ## {#float-builtin-functions}
+
+<table class='data'>
+  <thead>
+    <tr><td>Precondition<td>Built-in<td>Description
+  </thead>
+  <tr algorithm="scalar case, float abs">
+    <td>|T| is f32
+    <td class="nowrap">`abs(`|e|`:` |T| `) -> ` |T|
+    <td>(GLSLstd450FAbs)
+  <tr algorithm="vector case, float abs">
+    <td>|T| is f32
+    <td class="nowrap">`abs(`|e|`:` vec|N|<|T|> `) -> ` vec|N|<|T|>
+    <td>(GLSLstd450FAbs)
+  <tr algorithm="scalar case, acos">
+    <td>|T| is f32
+    <td class="nowrap">`acos(`|e|`:` |T| `) -> ` |T|
```

Comment:
do you want feedback on how these methods should be fixed to be portable, here? or would you prefer to discuss it separately? it seems like a mini-investigation is needed. A table of sorts, where at least we say what the expected input range is, and what kind of arguments are accepted

Reply:
I think that should come separately. This is already large, and I'd like to handle that in smaller CLs where we can focus on just one thing.

Reply:
Would you want to create a separate issue for filtering out this list, and analyzing the restrictions?
I'm just concerned that we'll land this, and then the whole list becomes a status quo.

Reply:
Are there ones that need to be removed? This list is already a subset of the current status quo, as GLSL.std.450 is said to be supported. This list is the subset of that that I've found I could easily (ie there is a builtin) translate into MSL, SPIR-V and HLSL.  The list in the first comment are the ones that I either didn't investigate yet (pack/unpack) or I didn't see a trivial translation.

We should file a bug once this lands for filling out the description and restrictions for each of these methods.